specification.md

```diff
@@ -103,11 +104,6 @@ Resource Managers (LRMs), such as PBS/Torque, SLURM, etc. To a first
 approximation, a job management API is understood as an abstraction layer
 on top of various LRMs.
 
-Job management is sometimes also provided by execution managers, with
-capabilities similar to LRMs but operating in user space, on a limited
-subset of resources such as within a job's allocation.  This job
-management API aims to also transparently abstract such execution managers.
-
```

Comment:
I think this paragraph is meaningful and correct.  Do we want to keep it?

Reply:
Agree and done.